aspnet/whitepapers/aspnet-and-iis6.md

```diff
@@ -177,25 +177,25 @@ To enable integrated authentication between SQL Server and ASP.NET, you will nee
 
 Open SQL Server Enterprise Manager (Start | Programs | Microsoft SQL Server | Enterprise Manager), select the appropriate server, and expand the Security folder:
 
-![](aspnet-and-iis6/_static/image22.jpg)
+![A screenshot of the Windows SQL Enterprise Manager screen. The  menu has Logins highlighted.](aspnet-and-iis6/_static/image22.jpg)
```

Comment:
A screenshot of the Windows S Q L Enterprise Manager screen. The menu has Logins highlighted.